lms/resources/_js_config/__init__.py

```diff
@@ -433,13 +433,8 @@ def _sync_api(self):
 
         req = self._request
 
```

Comment:
This isn't the only place in this file these values are used. They are also used in the file picker sections above, however I feel like it isn't correct to use the values there.

I think we want a similar abstract to `Product` for things like `FilePickerSource` which might fill out these values. The _might_ happen to match those for Blackboard in Blackboard files, but that feels like co-incidence.

For example if we used a 3rd party file picker like Google and integrated OAuth it would be a mistake to pass in the Blackboard OAuth settings just because you happen to be in Blackboard.